app/models/service.rb

```diff
@@ -12,11 +12,23 @@ def url
   end
 
   def down?
-    document = Nokogiri::HTML(open(url))
-    document.css('.page-status').text.include?('All Systems Operational') == false
+    !fetch_document.css('.page-status').text.include?('All Systems Operational')
   end
 
   def to_hash
     { id: id, name: name, url: url }
   end
+
+  private
+
+  def fetch_document
+    Nokogiri::HTML(open(url))
+  rescue RuntimeError => error
+    if error.message.include? 'redirection forbidden'
```

Comment:
I don't think so. I would rather see errors in failed jobs, otherwise we wouldn't know anything is wrong. I guess we could report issues to Sentry.